### ISSUE TITLE
Markers and Clusters are redraw when zoom is changing

### ISSUE THREAD

**Issue by aron-hivebrite** [googlemaps/js-markerclusterer#439]
I reviewed this example `http://jsfiddle.net/geocodezip/f5baj1uq/`
it uses `https://cdnjs.cloudflare.com/ajax/libs/markerclustererplus/2.1.4/markerclusterer.js`
I love it because I zoom markers and clusters and them are not redrawn.
[MarkerCluster-Ok.webm](https://user-images.githubusercontent.com/67704982/192104052-b6b67bc5-cc0b-44a8-bed3-1b862caaf7fe.webm)

But checking the last version `https://googlemaps.github.io/js-markerclusterer/public/defaults/` I see that all markers and clusters are drawn for example in each zoom changing giving a bad user experience because unnecessary markers and clusters are drawing again.
[MarkerCluster-Ko.webm](https://user-images.githubusercontent.com/67704982/192104133-b37be700-7e72-44f8-898c-b9b8bb91f1cb.webm)

Is there an option in the last version to avoid flash/redraw effect in markers and clusters? 
Thanks for your response.

**Comment by cernei** [comment]
True. Same happens when you dynamically add marker((  Everything re-renders.

**Comment by emilsrits** [comment]
Any solution to this issue? Experiencing the same problem - all markers non-clustered and clustered flicker on every zoom level change.

**Comment by andylacko** [comment]
in my case this flicker is blocking me from using components like **Popper** ( https://mui.com/material-ui/react-popper/ ) on **AdvancedMarkerElement** children, because it renders no element for a while and Popper jumps to left top corner and then back because **AnchorElement** is removed for a while.

I don't have very good experience with this lib for many many years.

BTW: Thank you so much whoever did that for implementing AdvancedMarkerElement, using HTML for markers saves ton of time and that marker works like charm so far
